File: python/aula1/exemplo1.py
import random
from python.util import popularListaArquivo

listUniqueNumbers = []

n = int(input("quantos numeros inteiro quer gerar? "))

for i in range(n):
    numero_sorteado = random.randint(0, 10)
    if numero_sorteado not in listUniqueNumbers:
        listUniqueNumbers.append(numero_sorteado)


listUniqueNumbers.sort()
print("Total de elementos:", len(listUniqueNumbers))
print(listUniqueNumbers)

File: csharp/aula1/Exemplo1.cs
using System;
using System.Collections.Generic;

class Exemplo1
{
    static void Main()
    {
        Random random = new Random();
        List<int> listUniqueNumbers = new List<int>();
        int num;

        System.Console.WriteLine("Quantos numeros inteiros quer gerar?");
        while (true) {
            if (int.TryParse(Console.ReadLine(), out num)) {
            break;
            }
            Console.WriteLine("Valor invalido! Digite apenas numeros.");  
        }
        System.Console.WriteLine($"Numeros para gerar: {num}");


        for (int i = 0; i < num; i++)
        {
            int numeroSorteado = random.Next(0, num);
            if (!listUniqueNumbers.Contains(numeroSorteado)) {
                listUniqueNumbers.Add(numeroSorteado);
            }
        }

        listUniqueNumbers.Sort();

        foreach (int n in listUniqueNumbers)
        {
            System.Console.Write(n + " ");
        }
    }
}